.vscode/settings.json

```diff
@@ -16,5 +18,23 @@
     ],
     "python.analysis.extraPaths": [
         "/vendor"
-    ]
-}
+    ],
+    "files.associations": {
+        "*.html": "jinja-html"
+    },
+    "emmet.includeLanguages": {
+        "jinja-html": "html"
+    },
+    "prettier.disableLanguages": [
+        "jinja-html"
+    ],
+    "beautify.language": {
+        "html": [
+            "jinja-html"
+        ]
+    },
+    "[jinja-html]": {
+        "editor.formatOnSave": true,
+        "editor.defaultFormatter": "HookyQR.beautify"
```

Comment:
Do we need to add this extension to the devcontainer config as well?

Reply:
Added in devcontainer